components/ord-service/src/main/java/com/sap/cloud/cmp/ord/service/storage/model/helperentities/DocumentationLabelEntityType.java

```diff
@@ -0,0 +1,27 @@
+package com.sap.cloud.cmp.ord.service.storage.model.helperentities;
```

Comment:
You need to add 5 more classes in the `helperentities`:

- `CorrelationIdEntityType`
- `ChangelogEntityType`
-  `LinkEntityType`
-  `EntityTypeProduct` due to the m2m relation (similar to `PackageProduct` for example)
-  `EntityTypeSuccessor`